_posts/2016-04-01-fintech-demo-day.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title: "FinTech Sandbox Demo Day"
+description: "A short recap of the FinTech Sandbox's Demo Day"
+category:
+tags: [FinTech, conference]
+author: vikram_ramakrishnan
+---
+{% include JB/setup %}
+
+Last week, [Greenfield](http://www.greenfieldhq.com) attended the [FinTech Sandbox](http://FinTechsandbox.org/)'s Demo Day, which highlighted
+some of the exciting FinTech startups which have gone through the program.
+The FinTech Sandbox connects some phenomenal data partners (Thomson Reuters,
+Factset, Tradier, Xignite, and others), accelerator partners and infrastructure
+partners (like AWS) with startups. The benefit here is that what was once very
+expensive to get started (due to the cost of data access) can now be prototyped
+and worked on quickly and for free.
+
+A couple high level themes arose from the event:
+
+### Technology is used for better decision making
+
+Each presenting startup implements technology to enable better decision making
+by its end customers. Some offer tools that allow users to have as little input
+as necessary to make a decision while others require significant input. However,
+they all fundamentally require inputs that get parsed, calculated and analyzed,
+which then is fed back to the user in a nice UI with some clarity to help the
```

Comment:
I wasn't at this event, but it seems like a point could possibly be made here about the value of design. As your point below illustrates, data alone is not useful unless its organized and presented in a way that humans can understand and act on. You kind of already say this, so I dunno if it's worth mentioning design. I'm just thinking about our trying to appeal to designers and showing that our company is one that values design and adds business value via thoughtful design.

I might also move reorder these two points, so that first you get the problem, i.e. data feeds are a headache. Then you get a solution, technology allows us to make sense of this data and make it useable.